ui/eureka/eureka_config.ts

```diff
@@ -193,5 +193,13 @@ UserConfig.registerOptions('eureka', {
         options['RefreshRateMs'] = seconds * 1000;
       },
     },
+    {
+      id: 'showBAPortalTimer',
+      name: {
+        en: 'Show Timer for BA red/blue Portals',
```

Comment:
I think it'd be fine to just always show portal timers.  I don't think this needs an option.

Reply:
Fair enough! I was 50/50 about it in the first place and only added it at the last minute.